tests/behat/theme_boost_union_contentsettings_footer.feature

```diff
@@ -19,18 +19,27 @@ Feature: Configuring the theme_boost_union plugin for the "Footer" tab on the "C
 
   Scenario: Setting: Footnote - Use the footnote setting to show a string in the page footer on the Dashboard, on the course pages and on the login page
     Given the following config values are set as admin:
-      | config   | value             | plugin            |
-      | footnote | Whatever footnote | theme_boost_union |
+      | config   | value                                                                                              | plugin            |
+      | footnote | <span lang="en" class="multilang">Footnote</span><span lang="de" class="multilang">Fussnote</span> | theme_boost_union |
+    And the "multilang" filter is "on"
+    And the "multilang" filter applies to "content and headings"
     When I log in as "admin"
     And I follow "Dashboard"
     Then "#footnote" "css_element" should exist
-    And I should see "Whatever footnote" in the "#footnote" "css_element"
+    And ".text_to_html" "css_element" should not exist in the "#footnote" "css_element"
+    And I should see "Footnote" in the "#footnote" "css_element"
+    And I should not see "<span lang=\"en\" class=\"multilang\">Footnote</span>" in the "#footnote" "css_element"
+    And I should not see "FootnoteFussnote" in the "#footnote" "css_element"
     And I log out
     And I log in as "teacher1"
     And I am on "Course 1" course homepage
     Then "#footnote" "css_element" should exist
-    And I should see "Whatever footnote" in the "#footnote" "css_element"
+    And I should see "Footnote" in the "#footnote" "css_element"
+    And I should not see "<span lang=\"en\" class=\"multilang\">Footnote</span>" in the "#footnote" "css_element"
+    And I should not see "FootnoteFussnote" in the "#footnote" "css_element"
     And I log out
     And I follow "Log in"
     Then "#footnote" "css_element" should exist
```

Comment:
Don't you want to add the
`And ".text_to_html" "css_element" should not exist in the "#footnote" "css_element"`
line once more here?